mitosheet/src/components/taskpanes/Download/DownloadTaskpane.tsx

```diff
@@ -207,13 +259,14 @@ const DownloadTaskpane = (props: DownloadTaskpaneProps): JSX.Element => {
                 <TextButton
                     variant='dark'
                     width='block'
-                    disabled={exportString === '' }
+                    error={!!invalidFileNameWarning}
+                    disabled={!!invalidFileNameWarning || exportString === '' }
                     href={exportHRef} 
                     download={exportName}
                     onClick={onDownload}  
                 >
                     
-                    {exportString === '' ? (<>Preparing data for download <LoadingDots /></>) : `Download ${props.uiState.exportConfiguration.exportType === 'csv' ? 'CSV file': 'Excel workbook'}`}
+                    {invalidFileNameWarning ? <>Invalid input.</> : exportString === '' ? (<>Preparing data for download <LoadingDots /></>) : `Download ${props.uiState.exportConfiguration.exportType === 'csv' ? 'CSV file': 'Excel workbook'}`}
```

Comment:
Maybe slightly more helpful: 

```suggestion
                    {invalidFileNameWarning ? <>{invalidFileNameWarning}</> : exportString === '' ? (<>Preparing data for download <LoadingDots /></>) : `Download ${props.uiState.exportConfiguration.exportType === 'csv' ? 'CSV file': 'Excel workbook'}`}
```

Reply:
I guess what I was thinking is that if we add any more validation that we'd want it to be more general (in case there were multiple validation errors). But now that I'm thinking about it, I do think that generally when a form is invalid, the submission button becomes disabled but the actual text stays the same, and then you'd have more info in a tooltip or add some red text above the submission button. Maybe we can brainstorm a little about the design